backend/models/note.ts

```diff
@@ -4,7 +4,6 @@ interface INote {
   id: string;
   content: string;
   important: boolean;
-  user: mongoose.Types.ObjectId;
```

Comment:
Why do we remove `user` from note? Isn't it that `note` should have ref to `user`?